packages/storybook/src/generators/configuration/project-files/.storybook/main.js__tmpl__

```diff
@@ -17,13 +17,35 @@ module.exports = {
     if (rootMain.webpackFinal) {
       config = await rootMain.webpackFinal(config, { configType });
     }
-    
+
     <% if (existsRootWebpackConfig) { %>// for backwards compatibility call the `rootWebpackConfig`
       // this can be removed once that one is migrated fully to
       // use the `webpackFinal` property in the `main.js` file
       config = rootWebpackConfig({ config });
     <% } %>
 
+    <% if (uiFramework === '@storybook/angular') { %>
+      /**
+       * Remove html raw loader that breaks Jit compilation as suggested here https://github.com/storybookjs/storybook/issues/16977#issuecomment-1004059631.
+       * The issue references:
+       * - https://github.com/storybookjs/storybook/issues/16977#issuecomment-1003399336
+       * - https://github.com/storybookjs/storybook/issues/16977#issuecomment-1004180729
+       * Here's what should be removed:
+       * {
+       *   test: /\.html$/,
+       *   loader: './node_modules/raw-loader/dist/cjs.js',
+       *   exclude: /\.async\.html$/
+       * },
+       */
```

Comment:
Do you think we could make this comment smaller, or maybe remove the github issue references/links, and just keep the explanation? (the Jit breaking, and what needs to be removed for example)?

Reply:
Sure, I will remove it in a while once I'm around my laptop.

Reply:
Sure, no rush! I'll take a look tomorrow my morning in any case! :D

Reply:
@mandarini I made the change you requested.